ckanext/reclineview/theme/public/vendor/recline/recline.js

```diff
@@ -2276,16 +2276,16 @@ my.Map = Backbone.View.extend({
 
   // Private: Sets up the Leaflet map control and the features layer.
   //
-  // The map uses a base layer from [MapQuest](http://www.mapquest.com) based
+  // The map uses a base layer from [Stamen](http://maps.stamen.com) based
   // on [OpenStreetMap](http://openstreetmap.org).
   //
   _setupMap: function(){
     var self = this;
     this.map = new L.Map(this.$map.get(0));
 
-    var mapUrl = "//otile{s}-s.mqcdn.com/tiles/1.0.0/osm/{z}/{x}/{y}.png";
-    var osmAttribution = 'Map data &copy; 2011 OpenStreetMap contributors, Tiles Courtesy of <a href="http://www.mapquest.com/" target="_blank">MapQuest</a> <img src="//developer.mapquest.com/content/osm/mq_logo.png">';
-    var bg = new L.TileLayer(mapUrl, {maxZoom: 18, attribution: osmAttribution ,subdomains: '1234'});
+    var mapUrl = "//stamen-tiles-{s}.a.ssl.fastly.net/terrain/{z}/{x}/{y}.jpg";
+    var osmAttribution = 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://creativecommons.org/licenses/by-sa/3.0">CC BY SA</a>.';
+    var bg = new L.TileLayer(mapUrl, {maxZoom: 18, attribution: osmAttribution});
```

Comment:
IIRC, the stamen project supports subdomains `a-d`, while leaflet defaults to `a-c`. Should we define the full range?